compiler/crates/relay-typegen/src/visit.rs

```diff
@@ -529,6 +539,13 @@ fn get_resolver_arguments(
                 optional: false,
             });
         }
+    } else if resolver_metadata.live {
+        resolver_arguments.push(KeyValuePairProp {
+            key: "key".intern(),
+            value: AST::RawType("void".intern()),
```

Comment:
Preferable to use the `intern!("void")` macro here and above. That creates the string once statically rather than needing to intern it each time this branch is run.